.github/workflows/packaging.yml

```diff
@@ -0,0 +1,36 @@
+name: Package
+# This workflow is triggered on pushes to the repository.
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  package:
+    name: Package and push
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Install bumpversion dependency
+        run: |
+          python -m pip install --upgrade pip
+          pip install bump2version pyyaml
+      - name: Bump version if necessary
```

Comment:
Since these same scripts are being used by all charts, what if we extracted them into a simple pip installable package? Can github actions be included through an external package too?

Reply:
I guess the consensus is that we will do this in a future refactoring run.